src/server/dispatcher.ts

```diff
@@ -70,7 +69,7 @@ export class Dispatcher {
   public constructor(
     registry: Registry,
     contextRegistry: ContextRegistry,
-    openApiDocument?: OpenApiDocument,
+    openApiDocument?: OpenApiDocument
```

Comment:
Sorry about all my linter-related changes. I can resubmit without those if you prefer.

Reply:
Ideally the linter wouldn't have made those kinds of changes in the first place. Did you run `yarn install` to make sure you have the latest version of Prettier? If that doesn't fix it, maybe we need to add a .prettierrc file to this repo?